File: packages/java/tests/spring/react-grid-test/src/main/java/dev/hilla/test/reactgrid/PersonService.java
package dev.hilla.test.reactgrid;

import dev.hilla.BrowserCallable;
import dev.hilla.crud.CrudRepositoryService;

import com.vaadin.flow.server.auth.AnonymousAllowed;

@BrowserCallable
@AnonymousAllowed
public class PersonService
        extends CrudRepositoryService<Person, Long, PersonRepository> {

}

File: packages/java/tests/gradle/kotlin-gradle-test/src/main/kotlin/com/vaadin/hilla/gradle/test/services/PersonService.kt
package com.vaadin.hilla.gradle.test.services

import com.vaadin.hilla.gradle.test.data.Person
import com.vaadin.hilla.gradle.test.data.PersonRepository
import com.vaadin.flow.server.auth.AnonymousAllowed
import com.vaadin.hilla.BrowserCallable
import com.vaadin.hilla.crud.CrudRepositoryService

@AnonymousAllowed
@BrowserCallable
class PersonService : CrudRepositoryService<Person, Long, PersonRepository>()
